R/zarr-array.R

```diff
@@ -1317,3 +1262,82 @@ ZarrArray <- R6::R6Class("ZarrArray",
 as.array.ZarrArray = function(x, ...) {
   x$as.array()
 }
+
+get_parallel_settings <- function(on_windows = (.Platform$OS.type == "windows"),
+                                  parallel_option = getOption("pizzarr.parallel_read_enabled", FALSE),
+                                  progress = getOption("pizzarr.progress_bar", FALSE)) {
```

Comment:
Since `pizzarr.progress_bar` is a new option being introduced here, can it also be added to https://github.com/keller-mark/pizzarr/blob/main/R/options.R for initialization and to support controlling via environment variable?

Reply:
This is done now.